templates/web/base/footer.html

```diff
@@ -25,7 +25,7 @@
                             %]span[% ELSE %]a href="[% base %]/my"[% END
                             %]>[% c.user_exists ? loc("Your account") : loc("Sign in") %]</[% c.req.uri.path == '/my' ? 'span' : 'a' %]></li>[%
                         %]<li><[% IF c.req.uri.path == '/reports'
-                            %]span[% ELSE %]a href="[% base %]/reports"[% END
+                            %]span[% ELSE %]a href="[% base %]/reports[% IF c.user_exists %]?filter_category=[% c.user.categories.join(",") | uri | html %][% END %]"[% END
```

Comment:
As with the front page field you've added, do you want to only output this extra part of there are categories to include?